test-app-sitebricks.yaml

```diff
@@ -0,0 +1,148 @@
+swagger: '2.0'
+info:
+  title: 'test-app-sitebricks'
+  description: 'simple test app with sitebricks'
+  version: '1.0-SNAPSHOT'
+host: "test-app-sitebricks.appspot.com"
+schemes:
+ - https
+basePath: "/"
+paths:
+  /users:
+    get:
+      summary: "returns all users registered in the system"
+      operationId: "getUsers"
+      produces:
+      - "application/xml"
+      - "application/json"
+      responses:
+        200:
+          description: "successfully return all users"
+    post:
+      summary: "register user in the system"
+      operationId: "registerUser"
+      consumes:
+      - "application/xml"
+      - "application/json"
+      produces:
+      - "application/xml"
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "user"
+        description: "user that should be added to the system"
+        required: true
+        schema:
+          $ref: "#/definitions/User"
+      responses:
+        201:
+          description: "user registered successful"
+        400:
+          description: "invalid consumed data"
+  /users/{userId}:
+    get:
+      summary: "find user by given id"
+      operationId: "findUserById"
+      produces:
+      - "application/xml"
+      - "application/json"
+      parameters:
+      - name: "userId"
+        in: path
+        description: "id of user to return"
+        required: true
+        type: "integer"
+      responses:
+        200:
+          description: "returns required user successful"
+        400:
+          description: "invalid id of user"
+    delete:
+      summary: "delete a user"
+      operationId: "deleteUserById"
+      produces:
+      - "application/xml"
+      - "application/json"
+      parameters:
+      - name: "userId"
+        in: path
+        description: "id of user that wants to delete"
+        required: true
+        type: "integer"
+      responses:
+        200:
+          description: "user was deleted successfully"
+        400:
+          description: "invalid user id"
+  /accounts/{userId}/balance:
+    get:
+      summary: "returns user account balance"
+      operationId: "getUserBalance"
+      produces:
+      - "application/xml"
+      - "application/json"
+      parameters:
+      - name: "userId"
+        in: path
+        description: "id of user that want to get balance"
+        required: true
+        type: "integer"
+      responses:
+        200:
+          description: "return balance of user successfully"
+        400:
+          description: "invalid user id"
+  /accounts/{userId}/deposit/{value}:
```

Comment:
in your other PR I've suggested to hide the value from the path and make it a request body parameter, is there any reason for you to think that it would be better to be in the path?